x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/settings_flyout/alerts_preview/index.test.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AlertConsumers } from '@kbn/rule-registry-plugin/common/technical_rule_data_field_names';
+import { SECURITY_SOLUTION_RULE_TYPE_IDS } from '@kbn/securitysolution-rules';
+import { render } from '@testing-library/react';
+import React from 'react';
+import * as uuid from 'uuid';
+
+import { AlertsPreview } from '.';
+import { useKibana } from '../../../../common/lib/kibana';
+
+jest.mock('uuid', () => ({
+  v4: jest.fn().mockReturnValue('mocked-uuid'),
+}));
+
+jest.mock('../../../../common/lib/kibana', () => ({
+  useKibana: jest.fn().mockReturnValue({
```

Comment:
Heads up, this refactor might cause issues with this test because `useKibana`/`trigger_actions_ui` dependency is removed from this file: https://github.com/elastic/kibana/pull/207878/files#diff-daba14cc816086a7b0e49baaa8e6e369672354782017dc9fe68d2be524c348e6L26